app/components/common/simple-modal.js

```diff
@@ -0,0 +1,3 @@
+import AuModal from '@appuniversum/ember-appuniversum/components/au-modal';
+
+export default class SimpleModal extends AuModal {}
```

Comment:
Well, the `AuModal` component contains a lot of UI elements which we do not want for this generic version, we could of course also not extend it and just re-implement it.

Reply:
if it's not too much trouble, I think re-implement may be better? we'd be depending quite heavily on the internal implementation of the AuModal component, so it might break in a minor bump of appuniversum

Reply:
Will do, should we also copy the css/stylesheets you reckon?